api/desecapi/models.py

```diff
@@ -926,7 +911,22 @@ class Kind(models.TextChoices):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if not self.content:
-            self.content = captcha_default_content(self.kind)
+            self.content = self.captcha_default_content(self.kind)
+
+    @staticmethod
+    def captcha_default_content(kind: str) -> str:
```

Comment:
If had meant making it an object method, and using `self.kind` ;-) But now we'll keep it where it was anyways, so all good.